_posts/2019-01-22-new.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Buildah V1.6 Release Announcement
+categories: [new]
+---
+Buildah V1.6 is here with lots of improvements and enhancements.  Check out the [Release Announcement](https://buildah.io/releases/2019/01/22/Buildah-version-v1.6.html) for all the details.
```

Comment:
Here's also a capital `V` (instead of a small one).

Reply:
Thanks @vrothberg , I'll touch them up.